File: Problem Solving/C/prime.c
/*check wheather a given number is prime or not*/
#include<stdio.h>
#include<conio.h>
void main()
{	
int num,count=0;
printf("enter your number");
scanf("%d",&num);
for(int i=1;i<=num;i++)
{
if(num%i==0)
{
count++;
}
}
if(count==2)
{
printf("%d is the prime number",num);
}
else
{
printf("%d is not the prime number",num);
}
getch();
}

File: Problem Solving/Python/prime.py
#check whether a given number is prime or not
count=0
print("Enter your number")
num=int(input())
for i in range(1,num+1):
    if(num%i==0):
        count=count+1
if(count==2):
    print(num,"is the prime number")
else:
    print(num,"is not the prime number")
